### PR TITLE
Feature/dts plugin

### PR DESCRIPTION
**Pull request by rafaelmaeuer**
- adding first draft of dts plugin (running in test so far)
- upgrading dependencies
- updating eventhub pubsub SA email

New local CLI tool to convert coreIds: `yarn coreId stringToConvert`

### CODE REVIEW

File: src/utils/plugins/dts/event.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+			headers: {
+				...defaultHeaders,
+				Authorization: pluginSecrets.credentials.dashboard,
+			},
+		}
+		const integrationsListAction = await fetch(lookupConfig.url, lookupConfig)
+		const integrationsList = await integrationsListAction.json()
+
+		//
+		// DEV catch empty list here
```
Comment (rafaelmaeuer) [inline src/utils/plugins/dts/event.js:55]: Should we implement this checks before PR merge?

File: src/utils/secrets/_client.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+	ard-eventhub
+	by SWR audio lab
+
+*/
+
+const { SecretManagerServiceClient } = require('@google-cloud/secret-manager')
+
+const client = new SecretManagerServiceClient({
+	projectId: process.env.GCP_PROJECT_ID,
```
Comment (rafaelmaeuer) [inline src/utils/secrets/_client.js:11]: what is the purpose of this secrets manager?

File: docs/QUICKSTART.md [hunk 3]
```diff
@@ -84,6 +84,7 @@ In your system for every new event, you might follow a workflow like this:
 
 ```js
 {
+   "type": "music",
```
Comment (rafaelmaeuer) [inline docs/QUICKSTART.md:87]: I would prefer `type` on top as it is a required value

File: src/ingest/README.md [hunk 4]
```diff
@@ -13,6 +13,7 @@ Several environment variables need to be set in `.env` config in order to run th
 - REQUIRED `GCP_PROJECT_ID` - which GCP project ID to use for Pub/Sub and Datastore requests
 - REQUIRED `FIREBASE_API_KEY` - corresponding `API_KEY` which matches the `GCP_PROJECT_ID`
 - REQUIRED `GOOGLE_APPLICATION_CREDENTIALS` - where the Google Cloud Service Account Key can be found (usually a path to a .json file)
+- REQUIRED `PUBSUB_SERVICE_ACCOUNT_EMAIL_INTERNAL` - for verification of internal publisher service account
```
Comment (rafaelmaeuer) [inline src/ingest/README.md:16]: as ingest doesn't start when this env isn't set it must be added as required env var in readme

File: src/utils/core/demo.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+
+
+const createHashedId = require('./createHashedId')
+
+
+const input = '28428'
```
[the remark this answers is not in the document]
Comment (frytg) [reply src/utils/core/demo.js]: see pull comment above, this was temporary for the new CLI tool

**Review by frytg — APPROVED** [review]

**Pull request merged**